### DISCUSSION TITLE
Getting to work

### DISCUSSION THREAD

**Discussion by abrookins** [aio-libs-abandoned/aioredis-py#963] [General]
Hey folks!

I'm Andrew Brookins - a developer advocate at Redis Labs.

A while back, I offered that Redis Labs could step in and take over maintainership, and now my schedule is finally open to start helping out. :)

Here are my current plans for the next few months:

* Get familiar with the library (I've used and contributed to redis-py, so this shouldn't be too hard!)
* Start reading through PRs and issues
* Add support for all Redis 6.2 features
* Push the v2 release forward by testing it in a production system (our full-text website search API)

@Andrew-Chen-Wang @seandstewart Let me know anything else I should look at or thoughts you guys have. I won't be full-time on this project, but intend to get the ball rolling from the Redis Labs side and act as a bridge to our client API developers.

**Comment by Andrew-Chen-Wang** [comment]
Hey @abrookins Good to hear! besides the given list, here are some perhaps unnecessary but additional tests that we could add:

 - [x] mypy check can be added somewhere in the linter. Maybe add it in the lint section of Makefile or disable the `needs: lint` portion from the CI? I wonder if people would be driven crazy by the linter CI.
 - [ ] ssl tests? Not sure https://github.com/aio-libs/aioredis-py/blob/82004b5fd148e2fb234a51c6966f45f50d1a7d93/Makefile#L62-L63
 - [ ] make sure docs are building correctly (this can either be via the PR webhook for RTD or just running make docs in a separate yaml file which detects when a path has been changed or not)

**Reply by abrookins** [reply]
Word! Great suggestions.

**Comment by Andrew-Chen-Wang** [comment]
I think it'd also be great to have an issue template for bugs setup, at least for alpha release.

**Reply by abrookins** [reply]
Yeah, for sure - I was just thinking that today, too.

**Comment by aviramha** [comment]
What about doing another release? Seems some bugs had been fixed.

**Reply by Andrew-Chen-Wang** [reply]
I'm going to be updating #1156 to be on par with Redis-py 4.0.1 tonight and then we can make a minor release.

**Comment by Andrew-Chen-Wang** [comment]
@abrookins @seandstewart should we make a wishlist issue of issues that should be resolved and pin it? For instance, we still haven't merged a solution for the lock issue #1040

**Reply by abrookins** [reply]
Folks, today is my last day at Redis (the company). I wish I could have contributed more to aioredis. I believe this library is a valuable and important part of the Redis ecosystem. I've been unable to recruit people internally -- there are only a few Python folks here, and we've all been hard at work on other projects.

And now my too-brief time has come to an end. I've added one of my colleagues to the project to keep the company's connection alive here, and I'll be watching!

**Reply by Andrew-Chen-Wang** [reply]
Thank you so much for the help and support @abrookins from the migration to all the bug and conceptual patches. This repository definitely would not be able to get so much done without you. I've learned so much about asyncio from your time assisting. Hope you all the best!
